com.inform7.IDE.appdata.xml

```diff
@@ -2,11 +2,28 @@
 <component type="desktop-application">
   <id>com.inform7.IDE</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <project_license>LicenseRef-Compiler-InformLicense-EverthingElseGPL-3.0</project_license>
+  <project_license>GPL-3.0</project_license>
```

Comment:
This looks good to me. The main thing I was concerned about is it showing up as "Proprietary" on Flathub, so since this is an actual [SPDX license expression](https://spdx.org/specifications), this problem should hopefully be fixed.